.gitpod.yml

```diff
@@ -1,7 +1,7 @@
 image:
   file: .gitpod.Dockerfile
 tasks:
-  - init: cargo install nu --all-features
+  - init: cargo install nu --features=user-visible
```

Comment:
I'd rather not do this because we've been teaching people to use `cargo install --all-features`